packages/react/package.json

```diff
@@ -48,6 +48,16 @@
 		"vitest": "^1.5.0"
 	},
 	"peerDependencies": {
-		"react": "^18.2.0"
+		"react": "^18.2 || ^19.0",
```

Comment:
I'm thinking we could even relax this further to `>=18` -- see e.g. https://github.com/chakra-ui/zag/blob/main/packages/frameworks/react/package.json#L45-L48